packages/internals/src/engine-commands/Library.ts

```diff
@@ -0,0 +1,35 @@
+import type { DataSource, GeneratorConfig } from '@prisma/generator-helper'
+
+/**
+ * Note: these types are copied from packages/engine-core/src/common/types/QueryEngine.ts
+ * to avoid circular type dependencies.
+ */
+
+export type GetConfigOptions = {
+  datamodel: string
+  ignoreEnvVarErrors: boolean
+  datasourceOverrides: Record<string, string>
+  env: NodeJS.ProcessEnv | Record<string, string>
+}
+
+export type ConfigMetaFormat = {
+  datasources: DataSource[]
+  generators: GeneratorConfig[]
+  warnings: string[]
+}
+
+export type Library = {
```

Comment:
I think this is now duplicated in `engine-core`, right? Could you please import this in `engine-core` in `type/Library` to deduplicate it?

Reply:
Before doing anything like this, we need to stop having `@prisma/internals` depend on `@prisma/engine-core`